dbt/models/qc/qc.vw_iasworld_asmt_all_with_prior_year_values.sql

```diff
@@ -0,0 +1,52 @@
+SELECT
+    legdat.parid,
+    asmt.taxyr,
+    legdat.user1 AS township_code,
+    pardat.nbhd,
+    pardat.class,
+    pardat.luc,
+    owndat.own1,
+    aprval.reascd,
```

Comment:
The raw `reascd` here is just a number, but it would be nice to make them more human-readable, e.g. `1: Assessor Correction`. Is the correct way to do this just to join to the `ccao.aprval_reascd` table?

Reply:
Should be just fine that way.